File: pkg/doctor/testdata/Pi.cs
using Stripe;

class Demo {
  void Go() {
    // payment_method_types in a comment must NOT match
    var options = new PaymentIntentCreateOptions
    {
      Amount = 1099,
      Currency = "eur",
      PaymentMethodTypes = new List<string> { "card", "ideal" },
    };
  }
}

File: pkg/doctor/testdata/pi.go
package main

import (
	"github.com/stripe/stripe-go/v79"
	"github.com/stripe/stripe-go/v79/paymentintent"
)

func main() {
	// payment_method_types in a comment must NOT match
	params := &stripe.PaymentIntentParams{
		Amount:             stripe.Int64(1099),
		Currency:           stripe.String("eur"),
		PaymentMethodTypes: stripe.StringSlice([]string{"card", "ideal"}),
	}
	pi, _ := paymentintent.New(params)
	_ = pi
}
